### ISSUE TITLE
Aesthetics

### ISSUE THREAD

**Issue by kamakshidasan** [QwertyWorks/qwertyworks.github.io#8]
- Would it be possible to add a scrolling effect when pressing the buttons at the top? Here is an example: http://www.christopher-ware.com/

- The different content tabs are not demarcated clearly. This could be done changing the background in each content tab and/or by adding a small heading/tag Here is the best example: https://www.apple.com/iphone-x/

**Comment by cruzer3008** [comment]
We added a different background color to every section...
Any suggestions regarding an alternate color scheme are most welcomed

**Comment by kamakshidasan** [comment]
Amazing stuff! This looks way better now! :)

**Comment by kamakshidasan** [comment]
I will leave this issue open -- it is always good to constantly improve the look+feel of the website.

**Comment by cruzer3008** [comment]
👍👍

**Comment by kamakshidasan** [comment]
The demarkation is missing between "choosing bulb" and the following section.

**Comment by kamakshidasan** [comment]
The demarkation is missing between "Control your lights with your voice!" and the following section.

**Comment by kamakshidasan** [comment]
Can you push the logo at the footer of the page to be aligned at the center instead?

<img width="1428" alt="screen shot 2018-08-07 at 17 06 33" src="https://user-images.githubusercontent.com/9362064/43784392-59bbeb7a-9a64-11e8-9e7c-c8b64f791a74.png">

**Comment by kamakshidasan** [comment]
Similarly, the other links at the masthead of the page. Can these be aligned at the center?

<img width="1335" alt="screen shot 2018-08-07 at 17 07 59" src="https://user-images.githubusercontent.com/9362064/43784504-956afc42-9a64-11e8-879d-821c2427db7d.png">

**Comment by kamakshidasan** [comment]
Masthead is missing on this page: http://joule.qwerty.works/Interns/aboutUs.html

**Comment by kamakshidasan** [comment]
Even on this page: http://joule.qwerty.works/Interns/contactUs.html

**Comment by maitri0411** [comment]
We haven't given masthead to these pages as "product" and "app" is a section on the index page and not a different page and thus the flow will not be appropriate.

**Comment by kamakshidasan** [comment]
Okay. You don't have to give a masthead, but the other links should be
definitely visible! In case the user is on the same section, and the user
clicks the same link on the masthead nothing should happen. In the other
cases, the page should simply smoothly scroll up and down.

**Comment by cruzer3008** [comment]
Demarcations added
Can you please explain where exactly should the logo be placed?
Something Like this ?? 
![43784392-59bbeb7a-9a64-11e8-9e7c-c8b64f791a74](https://user-images.githubusercontent.com/31630057/43827942-cb395120-9b18-11e8-84ea-97a79a507760.png)

**Comment by cruzer3008** [comment]
Please confirm if this is Okay..
Only then we will make changes to all pages 
![untitled](https://user-images.githubusercontent.com/31630057/43828927-67ec4afc-9b1b-11e8-986b-fdbb5a8c26b9.png)

**Comment by kamakshidasan** [comment]
Yes, a little more towards the left. There seems to be a lot of white space
to the left of the logo in comparison to the right of the "terms" column.

**Comment by cruzer3008** [comment]
Repositioned the Logo in the footer


Regarding the header....
The initial idea was to keep the anchor tags namely :- 
1)About Us - Which will lead to a different page
2)Product - Which will scroll down to the down on the same page 
3)App - Also scroll down to the same page
4)Contact - Lead to a different page
Product and app were kept on the same page due to the obvious reasons ( i.e ease of accessing that information and giving download links to the app)

The idea behind keeping headers on other pages was to prevent an abrupt start of the page and also clicking on the logo in the header would lead to index.html
Just to maintain the uniformity nothing was kept on the headers except the logo and companyName

We can make changes to the header on pages apart from index.html if you suggest (i.e what all can be added and how ?)
ONE suggestion would be to add icons of flipkart and amazon on the right side of the header on every page ( Not sure how advisable is that)

**Comment by kamakshidasan** [comment]
I understand your concern here. This solution below might work.

Place same header on all pages. If any button apart from homepage is clicked, perform:
1) Land on home page
2) Then slide down to the relevant section

Flipkart, Amazon: Ask @anant-QwertyWorks -- he will know better.

**Comment by kamakshidasan** [comment]
There seems to be a lot of whitespace above the heading "Meet the App!"

When I click the App button on the header, this makes the "Our Products" section to be "cut"

**Comment by kamakshidasan** [comment]
Also remove the full stop in the "Recreate ..."

What happened to warm white, cool white transitions?

Can you add the zoom transition effect on hovering in the "choose bulb" section, similar to the tiles below that?

**Comment by cruzer3008** [comment]
Hi,
We modified the NavBars on every page and yes it looks and feels comparitively better
(However there are a couple of pages wherein the elements in the list of the navbar which appear on pressing the hamburger icon on handheld devices are right aligned....We'll see to it)
Also removed the space above Meet the App

Regarding the issue of the " "Our Products" section to be "cut" "... We tried making the height of that section equal to the vertical height of the screen.But in that case there is a lot of blank space especially on 2560x1440 monitors
![untitled](https://user-images.githubusercontent.com/31630057/43887992-fc168b08-9bdd-11e8-8b9a-dfe38df57f4c.png)

Also we removed full stop from Recreate

Regarding the transitions on hover over the bulb we'll try and tell ( we'll have to learn some photoshop )

And also we added zoom transition effect on hover in the choose bulb section

**Comment by kamakshidasan** [comment]
Yo! Please replace the current orange green backgrounds on the webpage. Replace them with more visually appealing colours.

**Comment by kamakshidasan** [comment]
Can you push the navbar links, to be at the vertical center instead of bottom?

**Comment by kamakshidasan** [comment]
"Regarding the issue of the " "Our Products" section to be "cut" "... We tried making the height of that section equal to the vertical height of the screen.But in that case there is a lot of blank space especially on 2560x1440 monitors"

I understand. Just make sure that the space at the top of the section is also reflected at the bottom of the section.

Also, please do this for all sections.

**Comment by kamakshidasan** [comment]
Asking this again. 

In the section "Just say" add quotation marks along with the italic color text.
